src/cemerick/friend/util.clj

```diff
@@ -12,9 +12,14 @@
 ; this was never _really_ part of the API, was implemented (badly) before req/request-url was available
 (def ^:deprecated original-url req/request-url)
 
-(defn resolve-absolute-uri
-  [^String uri request]
-  (-> (original-url request)
-    java.net.URI.
-    (.resolve uri)
-    str))
```

Comment:
Should we be worried about an external client using this function?

Reply:
Perhaps it's safer to leave it and mark it as `deprecated`?